examples/version-migration/schema-upgrade/package.json

```diff
@@ -43,7 +43,7 @@
 		"@fluidframework/cell": "workspace:~",
 		"@fluidframework/container-definitions": "workspace:~",
 		"@fluidframework/container-loader": "workspace:~",
-		"@fluidframework/container-runtime": "workspace:^",
+		"@fluidframework/container-runtime": "workspace:~",
```

Comment:
This wasn't caught earlier because the version of syncpack we use doesn't handle the workspace protocol. The new version does, but it requires node20. See #22146.